src/js/app/index.js

```diff
@@ -237,6 +237,10 @@ var initDemo = function(sandbox) {
       'levels'
     ];
     commands = commands.join(';#').split('#'); // hax
+  } else if (params.hasOwnProperty('level')) {
+    commands = [
+      'level ' + unescape(params.level),
```

Comment:
I've not implemented checks on the `params.level` format/value here.

I'm not really confortable with this idea, as [an app must always validate input args/user values before using them](https://owasp.org/Top10/A03_2021-Injection/), but I've not implemented it myself to stick to the coding style for others parameters, where there is no check either.